documentation/source/user_section/tutorials/lesion-analysis/lesion-statistics/before-starting.rst

```diff
@@ -0,0 +1,22 @@
+Before starting this tutorial
+#############################
+
+You can continue this tutorial with your data (see :ref:`lesion-segmentation <lesion-segmentation>` for more information). Or with the provided example data, see below.
+
+#. If you want to continue the tutorial with the provided example data, make sure that you have the following files in your working directory:
+
+   * ``single_subject/data/t2_lesion/t2_seg.nii.gz``: A 3D binary mask of the spinal cord.
+   * ``single_subject/data/t2_lesion/t2_lesion.nii.gz``: A 3D binary mask of the lesion.
+
+   You can get these files by downloading :sct_tutorial_data:`data_lesion.zip`. TODO: add data_lesion.zip
+
+.. note::
+
+   Due to ethics, ``data_lesion.zip`` contains only spinal cord and lesion binary masks and not the T2w image used to generate them.
+
+
+#. Open a terminal and navigate to the ``single_subject/data/t2_lesion/`` directory:
+
+.. code:: sh
+
+   cd {PATH_TO_DOWNLOADED_DATA}/single_subject/data/t2_lesion/
```

Comment:
all that could be replaced with https://raw.githubusercontent.com/spinalcordtoolbox/spinalcordtoolbox/73ff5a3baeaff582d314a8320087d8e6dbf01ea3/documentation/source/user_section/tutorials/lesion-analysis/lesion-segmentation/before-starting.rst

Reply:
addressed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4586/commits/98bfc802e36db6174396ec12acb97f8cec2ffc0e